examples/fluent-bit/filter-multiline/README.md

```diff
@@ -0,0 +1,51 @@
+### FireLens Example: Concatenate Multiline or Stack trace log messages
+
+Multiline Filter is availble on `aws-for-fluent-bit` >= v2.18.0. It helps to concatenate messages that originally belong to one context but were split across multiple records or log lines. Common examples are stack traces or applications that print logs in multiple lines. More information could be found in [Fluent Bit official doc](https://docs.fluentbit.io/manual/pipeline/filters/multiline-stacktrace).
+
+With this filter, you are able to use Fluent Bit built-in parses with auto detection and multi format support on:
+- go
+- python
+- ruby
+- java (Google Cloud Platform Java stacktrace format)
+As those are built-in, you can directly specify them in a field called `multiline.parser` in `[FILTER]` section.
+ 
+As mentioned in the official doc, [Tail plugin](https://docs.fluentbit.io/manual/pipeline/inputs/tail#multiline-support) is recommended for doing the concatenation on messages split originally by Docker or CRI container engines. We will use it in our example below.
+
+Let's go through an example shows how to use the multiline filter:
+
+1. In order to parse a log file, you need to create one called `test.log` which contains some full lines, a custom Java stacktrace and a Go stacktrace. 
+2. Create another file `parsers_multiline.conf` which defines a multiline parser for custom stacktrace:
+```
+[MULTILINE_PARSER]
+    name          multiline-regex-test
+    type          regex
+    flush_timeout 1000
+    #
+    # Regex rules for multiline parsing
+    # ---------------------------------
+    #
+    # configuration hints:
+    #
+    #  - first state always has the name: start_state
+    #  - every field in the rule must be inside double quotes
+    #
+    # rules |   state name  | regex pattern                  | next state
+    # ------|---------------|--------------------------------------------
+    rule      "start_state"   "/(Dec \d+ \d+\:\d+\:\d+)(.*)/"  "cont"
+    rule      "cont"          "/^\s+at.*/"                     "cont"
+```
+3. Add your own custom config to `extra.conf`. In this config, you need to specify the above parser file in `[SERVICE]` section and have another `[FILTER]` section to add the parser. Note that `go` is a built-in parser so you can directly add it here. Also, this config includes your input log file.
+4. Build a custom Fluent Bit image using the provided Docker file (which simply copies those three customized files into the AWS for Fluent Bit image). You can place this extra config file anywhere in the Docker image *except* `/fluent-bit/etc/fluent-bit.conf`. That config file path is the path used by FireLens. Push this custom image to Amazon ECR.
```

Comment:
Do we want to provide the command or a link to build the Dockerfile?

Reply:
>Push this custom image to Amazon ECR.
 Also, can we add a link which explains how to push an image to ECR? 
 
 They can also push it to DockerHub which is easier for some customers, right?

Reply:
yeah we can add it

Reply:
As it is from our end, it might be better for us to use Amazon ECR?